MailCore/Cache/MainViewState.swift

```diff
@@ -28,7 +28,13 @@ public class MainViewState: ObservableObject {
     ///
     /// The selected thread is the last in collection, by convention.
     @Published public var threadPath = [Thread]()
-    @Published public var selectedFolder: Folder
+    @Published public var selectedFolder: Folder {
+        didSet {
+            Task.detached { [uid = selectedFolder.remoteId, name = selectedFolder.name] in
```

Comment:
This works, but we might want to have a shared private method in `SentryDebug` that do a `addBreadcrumb` in a task.